test/runtest.py

```diff
@@ -378,6 +378,28 @@ def f():
     assert ray.get(f.remote()) == ((3, "string1", Bar.__name__), "string2")
 
 
+def test_serialization_final_fallback(ray_start):
+    pytest.importorskip('catboost')
+    from catboost import CatBoostClassifier
+
+    @ray.remote
+    def fit(model):
+        train_data = np.random.randint(0, 100, size=(100, 10))
+        train_label = np.random.randint(0, 2, size=100)
+        model.fit(train_data, train_label, cat_features=[0, 2, 5])
```

Comment:
How long does this take?

Reply:
Less than 1 sec.

Reply:
Why not just make this test `ray.get(ray.put(CatBoostClassifier()))`?